app/models/conference.rb

```diff
@@ -4,6 +4,14 @@ class Conference < ActiveRecord::Base
 
   has_many :conference_preferences, dependent: :destroy
   has_many :attendees, through: :conference_preferences, source: :team
+  REGION_LIST = [
```

Comment:
Can you add another empty line before the constant please? Also, it's better to define constants at the very top of a class.